app/assets/stylesheets/theme/_variables.scss

```diff
@@ -34,6 +34,29 @@ $brand-info:            #9C27B0 !default;
 $brand-warning:         #ff9800 !default;
 $brand-danger:          #e51c23 !default;
 
+$books-clear:           #f3faee !default;
```

Comment:
Very small nitpick: rather than naming it `-clear`, perhaps `-light` might be a good name? Clear implies that it is transparent.